src/app.ts

```diff
@@ -1,5 +1,5 @@
-import path from 'path'
-import fs from 'fs'
+import { readFileSync, existsSync } from 'fs'
```

Comment:
We had this before and I'd like to keep this as `import fs from 'fs'` as it's more common to use the module name for nodejs apis.